go/tasks/plugins/webapi/agent/config.go

```diff
@@ -10,6 +10,8 @@ import (
 )
 
 var (
+	defaultTimeout = config.Duration{Duration: 10 * time.Second}
```

Comment:
do we want to make this configurable?

Reply:
Good question. I thought about how to deal with the default here. My reasoning is that if user bothers configuring this, they might just configure for each operation instead because it doesn't save much effort. That being said, I don't have a strong opinion and I can make it configurable for sure.